packages/launch/src/focused-launch/success/style.scss

```diff
@@ -103,3 +103,19 @@
 		width: auto;
 	}
 }
+
+.focused-launch-success__go-back-button {
+	color: var( --studio-gray-50 );
+}
+
+.focused-launch-container .action-buttons:not( .is-sticky ) {
```

Comment:
Probably these styles should belong to the [`packages/launch/src/focused-launch/summart/style.scss`](https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/focused-launch/summary/style.scss) file?

Also, these styles may not be necessary (see https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/focused-launch/summary/style.scss#L197-L207)